en/material-system/pass-parameter-list.md

```diff
@@ -43,7 +45,17 @@ This property doesn't exist by default, which means the pass is executed uncondi
 Specifies the rendering priority of the current pass, the bigger the number, the lower the priority. The default is (128), min is (0), max is (255), arithmetic operations between these constants and integer constants are supported.
 
 ## Stage
-Specifies which render stage the current pass belongs to.
+Specifies which render stage the current pass belongs to. For the built-in forward pipeline, the only available stage is 'default'.
+
+## Phase
+Specifies which phase the current pass belongs to. For the built-in forward pipeline, the available phases are 'default', 'forward-add' and 'shadow-caster'.
```

Comment:
'default', 'forward-add' and 'shadow-caster'.
-->
`default`, `forward-add` and `shadow-caster`.